packages/network-contracts/test/hardhat/OperatorTokenomics/Operator.test.ts

```diff
@@ -110,19 +110,20 @@ describe("Operator contract", (): void => {
             // - There is one single delegator with funds of 1000 DATA and no delegations.
             await setTokens(delegator, "1000")
             await setTokens(sponsor, "2000")
-            await setTokens(operatorWallet, "0")
+            await setTokens(operatorWallet, "1") // operator must self-delegate at least minDelegationWei to accept external delegations
```

Comment:
with the change from >= to > in the delegation policy the requirement changed from "at least" to "more than" I think
```suggestion
            await setTokens(operatorWallet, "1") // operator must self-delegate more than minDelegationWei to accept external delegations
```

Reply:
The "at least" refers to self delegation amount which can be equal to `minDelegationWei`. 
The ">= to >" refers to operator owner having "some" tokens, which could be 1 Wei (but that's not possible since a delegation must be >= minDelegationWei)

Reply:
ah right 👍